drivers/clock_control/renesas/r8a77951-cpg-mssr.c

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2022 IoT.bzh
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT renesas_r8a7795_cpg_mssr
+
+#include <errno.h>
+#include <soc.h>
+#include <drivers/clock_control.h>
+#include <drivers/clock_control/renesas-cpg-mssr.h>
+#include <dt-bindings/clock/renesas/renesas-cpg-mssr.h>
+#include <dt-bindings/clock/renesas/r8a7795-cpg-mssr.h>
```

Comment:
```suggestion
#include <zephyr/drivers/clock_control.h>
#include <zephyr/drivers/clock_control/renesas-cpg-mssr.h>
#include <zephyr/dt-bindings/clock/renesas/renesas-cpg-mssr.h>
#include <zephyr/dt-bindings/clock/renesas/r8a7795-cpg-mssr.h>
```

Reply:
Thanks, forgot to migrate to the new prefix !

Reply:
Migrated to new prefixes !